Components/navbar.css

```diff
@@ -30,11 +24,108 @@
     font-family: inherit;
     padding: 0.5rem;
     background-color: transparent;
-    width: 25vw;
-    max-width: 600px;
+    max-width: 600px !important;
```

Comment:
You can think of avoiding important, as its not considered as a best practice

Reply:
Removed. Thank you.